content/en/users/aai/check-in/vos/_index.md

```diff
@@ -284,40 +284,242 @@ From the steps defined [above](#managing-cou-admin-members):
 
 ## VO membership API
 
+### API v2
+
+Check-in provides a REST API that allows clients to manage membership
+information only for the VOs they are authoritative for.
+
+Features:
+
+- Members of the VO are identified via their EGI Check-in Community User Identifier (CUID)
+- Membership can be limited to a specified period
+- All [REFEDS](https://wiki.refeds.org/display/STAN/eduPerson+2020-01#eduPerson202001-eduPersonAffiliation) membership affiliations are supported
+- Role titles are supported
+- Different membership status values are supported, namely `Active`, `Expired`,
+  `Deleted`, `Suspended`
+- Check-in automatically changes the membership status from `Active` to
+  `Expired` beyond the validity period
+
+#### Connection Parameters
+
+<!-- prettier-ignore -->
+{{< tabpane >}}
+{{< tab header="Production environment" lang="shell" >}}
+# Export VO API Base URL parameter
+$ export VO_API_BASE_URL=https://aai.egi.eu/api/v2/VoMembers
+# Export CO ID parameter.
+# The CO ID is the number part of the API username prefix. 
+# e.g. for the username `co_2.test`, the CO_ID is `2`
+$ export CO_ID=2
+{{< /tab >}}
+{{< tab header="Demo environment" lang="shell" >}}
+# Export VO API Base URL parameter
+$ export VO_API_BASE_URL=https://aai-demo.egi.eu/api/v2/VoMembers
+# Export CO ID parameter.
+# The CO ID is the number part of the API username prefix.
+# e.g. for the username `co_2.test`, the CO_ID is `2`
+$ export CO_ID=2
+{{< /tab >}}
+{{< tab header="Development environment" lang="shell" >}}
+# Export VO API Base URL parameter
+$ export VO_API_BASE_URL=https://aai-dev.egi.eu/api/v2/VoMembers
+# Export CO ID parameter.
+# The CO ID is the number part of the API username prefix. e.g.
+# for the username `co_2.test`, the CO_ID is `2`
```

Comment:
Better splitting before not after `e.g.`, like for other lines.

Reply:
Done